test/EFCore.PG.FunctionalTests/Query/SimpleQueryNpgsqlTest.cs

```diff
@@ -15,7 +15,7 @@ public SimpleQueryNpgsqlTest(NorthwindQueryNpgsqlFixture<NoopModelCustomizer> fi
             : base(fixture)
         {
             Fixture.TestSqlLoggerFactory.Clear();
-            //Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
+            Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
```

Comment:
Comment it back (:

Reply:
Thanks, I get yelled at all the time at EF Core too...